tests/app/dao/test_notification_dao.py

```diff
@@ -1302,15 +1313,20 @@ def test_dao_timeout_notifications_doesnt_affect_letters(sample_letter_template)
     assert updated == 0
 
 
-def test_should_return_notifications_excluding_jobs_by_default(notify_db, notify_db_session, sample_service):
+def test_should_return_notifications_excluding_jobs_by_default(
+    notify_db,
+    notify_db_session,
+    sample_service,
+    sample_api_key,
+):
     assert len(Notification.query.all()) == 0
 
     job = sample_job(notify_db, notify_db_session)
     with_job = sample_notification(
-        notify_db, notify_db_session, created_at=datetime.utcnow(), status="delivered", job=job
+        notify_db, notify_db_session, created_at=datetime.utcnow(), status="delivered", api_key_id=None
     )
     without_job = sample_notification(
-        notify_db, notify_db_session, created_at=datetime.utcnow(), status="delivered"
+        notify_db, notify_db_session, created_at=datetime.utcnow(), status="delivered", api_key_id=sample_api_key.id
     )
```

Comment:
can you make this have three notifications? a job, an api, and a one-off? with their job/api_key layout as we'd expect to see in real life